ERCS/erc-7752.md

```diff
@@ -0,0 +1,937 @@
+---
+eip: 7752
+title: Restricted Security Token
+description: Restricted security token (private equity, debt, and derivative securuties)
+author: Matt Rosendin (@mrosendin) <matt@capsign.com>
+discussions-to: https://ethereum-magicians.org/t/erc-7752-equity-token/20735
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-08-06
+requires: 173
+---
+
+## Abstract
+
+An ERC token standard representing restricted securities. This new interface standardizes equity and alternative asset management for issuers, investors, transfer agents, and financial intermediaries, including private equity, debt, and derivatives.
```

Comment:
You should definitely define "restricted securities" here for anyone who might not know what that means. Your abstract should also contain a high level overview of the technical aspects of your proposal.